regulament_pregatit_feedback.txt

```diff
@@ -660,6 +665,79 @@ Art. 106.
+
+Art. 107+1. Acreditarea poate fi ridicată mai repede de termenul de doi ani dacă Centrul Local nu mai îndeplinește criteriile de acreditare.
+Art. 107+1. Numărul de Grupuri de Cercetași pe care un Centru Local le poate mentora simultan este specificat în IEDLOC.
+
+
+E2. Grupul de cercetași
+=======================
+
+Art. 107+1. Alcătuirea Grupului de Cercetași urmărește îndeaproape organizarea de Centru Local, definită la Art. 77.
+
+Art. 107+2. În grupurile de Cercetași, una din unități (sau unitatea, în cazul în care exită doar una) poate fi, prin excepție, formată dintr-o singură patrulă. Liderul care îndrumă unitatea trebuie să fie lider ONCR, dar poate să nu facă parte din Grupul de Cercetași; este ales de către membrii unității, în baza planului de mentorat și a propunerilor făcute de structura mentor.
+
+Art. 107+3. Grupurile de Cercetași există exclusiv sub mentoratul și gestiunea administrativă a unei structuri mentor, care de regulă este un Centru Local care îndeplinește condițiile minime pentru această situație. 
+
+Art. 107+4. În situații excepționale, în care nu este posibilă arondarea la un Centru Local din proximitatea geografică, Consiliul Director poate împuternici orice alt Centru Local eligibil să mentoreze grupul de cercetași, cu acordul acelui Centru Local. Alternativ, Consiliul Director poate împuternici nominal un grup de minim 3 lideri din ONCR care să asigure mentoratul grupului de cercetași, acesta primind formal mentoratul Consiliului Director; într-o astfel de situație excepțională, liderii și membrii grupului de cercetași vor participa la activități cercetășești și administrative în Centrele Locale ale liderilor desemnați ca mentori.
+
+Art. 107.5. Pentru înființarea unui Grup de Cercetași sunt necesare:
+	(1) Acordul unui grup informal de persoane de a forma împreună un Grup de cercetași
+	(2) Elaborarea, de către o structură mentor alături de Grupul de cercetași a unui Plan de Mentorat pe o perioadă de 1 an, care să descrie în ce condiții se va desfășura mentoratul și care conține minim:
+		(a) angajamentul reciproc al Grupului de Cercetași și al structurii mentor pentru dezvoltarea cercetășiei în Grupul de Cercetași, 
+		(b) obiective și un plan de acțiune concret cu privire la participarea Grupului de cercetași la activități cercetășești și organizatorice ale structurii mentor, precum și activitățile proprii Grupului de cercetași
+		(c) termenele pentru planificarea și evaluarea planului de mentorat
+		(d) persoanele desemnate din partea Grupului de cercetași și a stucturii mentor pentru urmărirea implementării planului de mentorat
+	(3) Acordul unei structuri de a prelua mentoratul Grupului de cercetași, în baza Planului de mentorat de la punctul (2), în condițiile art. 107+3 și 107+4; în situația din art. 107+3, acordul este dat printr-o hotărâre de Adunare Generală; în situația art. 107.4, acordul este dat printr-o decizie a Consiliului Director.
```

Comment:
Nu știu dacă ”în baza Planului de mentorat” e cea mai bună formulare. Că acordul de a mentora un GE e dat de AGul local întâi și apoi decizia finală e la CD (pe baza planului de mentorat). Sau e posibil să fi înțeles eu greșit flow-ul.

Reply:
Nivelul local trebuie să aibă un cuvânt de zis cu privire la Planul de Mentorat, pentru că îi afectează în mod direct (adică daca grupul nou va participa în campul centrului local, aia poate fi o schimbare majoră de perspectivă pentru grupul mentor ...). 

Cred că modul în care ar trebui să se întâmple lucrurile este că se face grupul nou în discuții cu niște oameni din grupul mentor, CCL are un acord de principiu (formal sau nu) și apoi cele doua grupuri (nou și mentor) lucrează la o propunere pe care să o prezinte AG local.

Reply:
atunci hai să menționăm la începutul frazei Acordul de a prelua ...., că e definit ce structură face asta mai apoi (că sună foarte general acordul unei structuri și pe mine personal m-a băgat în ceață întâi). Și aș menționa AG local, nu doar AG (ca să nu se înțeleagă AG ONCR).

Reply:
Done